CorsixTH/Lua/hospital.lua

```diff
@@ -1042,9 +1042,9 @@ end
 
 -- Creates complete emergency with patients, what disease they have, what's needed
 -- to cure them and the fax.
---!return outcome (bool), err (string) If emergency was scheduled, and if not what the error was
+--!return err (string) If emergency failed what was the issue
```

Comment:
Alternative formulation:  `--!return (optional string) Textual reason for failure, else nil`

The doc string is intended for persons only reading the function header, but not the code. Those persons don't know "err", they only want to know what the return value means.
Added "optional" to express that you sometimes don't return a string.